src/entity_recognizer/EntityFeatures.cpp

```diff
@@ -24,6 +24,7 @@
 void EntityFeatures::AddUnigramFeatures(SequenceInstanceNumeric *sentence,
```

Comment:
Apparently, that's not a mode that MVS accepts. 
It accepts spaces around conditional operators (like this "bool positive = a > 0 ? true : false;") or without them (like this "bool positive = a > 0?true:false;").